docs/40_node-operation/10_getting-started/50_snapshots.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Snapshots
+---
+
+A snapshot is a capture of the blockchain state at a specific point in time. It includes the sum of all changes that 
+occurred up until that point from transaction executions. This means that it includes all created accounts, contract code,
+contract data, and anything else that was created or modified on the blockchain.
+
+It does not however include the blockchain history. This means that it does not include the transactions themselves. These
+are stored in the `blocks.log` file. 
+
+## Syncing a fresh node
+
+To speed up the syncing process, you can download a snapshot and import it into your node. This will allow you to skip
+parts of the process that would otherwise take longer.
+
+### Required configuration
+
+You should have basic the following plugins enabled in your `config.ini` file:
+
+```ini
+plugin = eosio::chain_plugin
+plugin = eosio::chain_api_plugin
+plugin = eosio::net_plugin
+plugin = eosio::net_api_plugin
+plugin = eosio::producer_plugin
+plugin = eosio::producer_api_plugin
+plugin = eosio::state_history_plugin
+```
+
+### Getting a snapshot
+
+Below are some sites where you can download a recent snapshot:
+
+- [EOS Nation](https://snapshots.eosnation.io/)
+- [EOS Sweden](https://snapshots-main.eossweden.org/)
+
+### Before you start
+
+You should clear out your node's `data/state` directory. 
+
+### Importing the snapshot
+
+Now you can import the snapshot into your node:
+
+```shell
+nodeos --snapshot /path/to/snapshot.bin
+```
+
+> ⚠ **Warning**
+> 
+> Do not stop the node until it has received at least 1 block from the network, or it won't be able to restart.
+
+### If your node fails to receive blocks
+
+If nodeos fails to receive blocks from the network, then try using `cleos net disconnect` 
+and `cleos net connect` to reconnect nodes which timed out.
+
+> ⚠ **Warning**
+> 
+> Caution when using net_api_plugin. Either use a firewall to block access to your `http-server-address`, or change 
+> it to localhost:8888 to disable remote access.
+
+### Using a database filler
+
+If you are using a database filler, you need to look for `Placing initial state in block <block_num>` in the log. 
+
+Then you can start a filler with the following arguments:
+```shell
+... --fpg-create --fill-skip-to <block_num> --fill-trim
+```
+
+> On subsequent runs, you should not use the `--fpg-create` and `--fill-skip-to` arguments.
+
```

Comment:
Maybe add a short table of contents here for user to quickly select the procedure they want, as follows:

Select the procedure you need:
\- \[Create a snapshot with full state history](#creating-a-snapshot-with-full-state-history)
\- \[Restore a snapshot with full state history](#restoring-a-snapshot-with-full-state-history)
\- etc...

Reply:
They have this already available on the sidebar in the UI, so we don't need to embed these in the docs anymore and bloat them